### ISSUE TITLE
Make cloudamqp_custom_domain support custom certificate

### ISSUE THREAD

**Issue by FelipeEmerimRandon** [cloudamqp/terraform-provider-cloudamqp#298]
In the cloudamqp portal, we have an option called "Bring your own certificate" in the custom domain page. The Terraform provider does not expose a way to achieve the same configuration as far as we know. We are very interested in that feature as our use case requires that we provide our own certificate.

 If there is already a way to do this using terraform and we have not seen it, please disregard this issue.

**Comment by dentarg** [comment]
No, this is currently not possible as it isn't even exposed in our API. It is in our backlog of things to add but it has not been prioritized.
